portfolio/src/main/webapp/index.html

```diff
@@ -1,139 +1,207 @@
-    
+    <div class="parallax2"></div>
 
-        <!--Art photo gallery grid -->
-        <div class="gallery">
-        <a target="_blank" href="/images/blueflowers.jpg">
-        <img src="/images/blueflowers.jpg" alt="Blue Flowers" width="600" height="400">
-        </a>
+    <div style="color: #777;background-color:white;text-align:center;padding:50px 80px;text-align: justify;" id = "about">
+        <h3 style="text-align:center;">
+            <h3> About </h3>
+            <p> My name is Sarah Abdulkerim. I am a student studying Computer Science at Wellesley College.
+                I have experience programming in Python, C, and Java. As a hobby, I enjoy drawing with 
+                graphite and colored pencil media. I am passionate about software development and am interested 
+                in contributing and broadening both my back-end and front-end skills.  
+            </p>
+            <a href = '#cs'><button>Computer Sciene Portfolio!</button></a>
+        </h3>
     </div>
 
-        <div class="gallery">
-        <a target="_blank" href="/images/fluffy.jpg">
-        <img src="/images/fluffy.jpg" alt="Blossomed Blue Flower" width="600" height="400">
-        </a>
-    </div>
+    <div class="parallax3"></div>
 
-    <div class="gallery">
-        <a target="_blank" href="/images/pinkflowers.jpg">
-        <img src="/images/pinkflowers.jpg" alt="Pink Flowers" width="600" height="400">
-        </a>
-    </div>
+    <div style="color: #777;background-color:white;text-align:center;padding:50px 80px;text-align: justify;" id ="cs">
+        <h3 style="text-align:center;">
+            <h3> Computer Science Portfolio </h3>
+            <h2> Coursework: </h2>
+            <p> Computer Programming & Problem Solving, Computation Structures, Real-Time Systems and Software, Data Structures, Combinatorics & Graph Theory, Introduction to Algorithms</p>
+            <p> Programming Languages: Python, Java, and C </p>
+            <h2> Projects: </h2>
+            <div class="map" id = "MIT"></div>
+            <div class="map" id = "Wellesley"></div>
+            <!-- Container for the image gallery -->
+                <div class="container">
+
+                    <!-- Full-width images with number text -->
+                    <div class="mySlides">
+                        <div class="numbertext">1 / 2</div>
+                        <img src="/images/RAT.JPG" style="width:50%">
+                    </div>
 
-    <div class="gallery">
-        <a target="_blank" href="/images/pinkrose.jpg">
-        <img src="/images/pinkrose.jpg" alt="Pink Rose" width="600" height="400">
-        </a>
+                    <div class="mySlides">
+                        <div class="numbertext">2 / 2</div>
+                        <img src="/images/embed.gif" style="width:50%">
+                    </div>
+
+                    <!-- Next and previous buttons -->
+                    <a class="prev" onclick="plusSlides(-1)">&#10094;</a>
```

Comment:
I'm not really sure what's happening but the "right" arrow only seems to appear for one of the slides and the "left" arrow is placed in the middle of the picture.